tle/cogs/hard75Challenge.py

```diff
@@ -254,7 +254,9 @@ async def completed(self, ctx):
         else:
             current_streak=0
         if(current_streak==0):      # on first day!
-            current_streak=1    
+            current_streak=1   
+        if current_streak % 75 == 0:
+            await ctx.guild.get_member(user_id).add_roles(discord.utils.get(ctx.guild.roles, name = "Hard 75 Completed"))
```

Comment:
looks good! however can we change the name to Hard75X1? so that tag is small enough and also allows us to use X2 and X3 in future. also do you mind to test it first with maybe 65 instead of 75? so that if something breaks then we have time to repair it ?

Reply:
okay, I'll change the role name, and yes, I've tested it with 1 day streak in my private server.